proctor-common/src/main/java/com/indeed/proctor/common/ProctorUtils.java

```diff
@@ -97,7 +97,7 @@ public static Map<String, ValueExpression> convertLocalContextToValueExpressionM
             @Nonnull final Map<String, Object> values) {
         final Map<String, ValueExpression> context =
                 convertToValueExpressionMap(expressionFactory, values);
-        context.put(
+        context.putIfAbsent(
```

Comment:
why do we need to use putIfAbsent?

Reply:
Discovered in refactoring that this is needed. was adding the context variable as always true for range selector

Reply:
Reverted to put and refactored some. 

This function should just be used for adding in the variable to do verification of Test definitions when loading or testing changes in webapp.